packages/pigeon/example/app/macos/Runner/messages.g.m

```diff
@@ -219,10 +219,32 @@ - (void)flutterMethodAString:(nullable NSString *)arg_aString
           @"dev.flutter.pigeon.pigeon_example_package.MessageFlutterApi.flutterMethod"
              binaryMessenger:self.binaryMessenger
                        codec:PGNMessageFlutterApiGetCodec()];
-  [channel sendMessage:@[ arg_aString ?: [NSNull null] ]
-                 reply:^(id reply) {
-                   NSString *output = reply;
-                   completion(output, nil);
-                 }];
+  [channel
+      sendMessage:@[ arg_aString ?: [NSNull null] ]
+            reply:^(NSArray<id> *reply) {
+              if (reply != nil) {
```

Comment:
Can we extract this entire block to a static helper method (which we write conditionally if any FlutterAPI methods exist, to avoid unused method compilation issues)?

Reply:
Oh, the extraction part gets complicated if we do that though, right? In which case we can punt on that.